src/components/utils/common/CommonStyles.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+import { DescriptionStyle } from "./interface";
+
+export const colors = {
+  primary_black: "black",
+  primary_yellow: "rgba(232,196,122,0.8)",
+  primary_grey: "#fafafa",
+};
+
+export const Description = styled.span<DescriptionStyle>`
```

Comment:
Nice job
always use interfaces to pass props into styled-components,
It helps to the team to keep tracking the code